.github/release-drafter.yml

```diff
@@ -0,0 +1,44 @@
+# Config for https://github.com/apps/release-drafter
+name-template: 'v$NEXT_MINOR_VERSION (🌈 Insert Release Name)'
+tag-template: 'v$NEXT_MINOR_VERSION'
+categories:
+  - title: 'Dependency Updates'
+    label: 'dependencies'
+  - title: '🚀 Features'
+    label: 'feature'
```

Comment:
same here

Reply:
Not sure what you mean by `same here` ?

Reply:
To also have support for code quality branches